src/main/scala/mesosphere/marathon/core/externalvolume/impl/providers/DVDIProvider.scala

```diff
@@ -182,7 +182,8 @@ private[impl] object DVDIProviderValidations extends ExternalVolumeValidations {
         volume.external.size is isTrue("must be undefined for Docker containers")(_.isEmpty)
         volume.containerPath is notOneOf(DotPaths: _*)
         // TODO(jdef) change this once docker containerizer supports relative containerPaths
-        volume.containerPath should matchRegexFully(AbsolutePathPattern)
+        volume.containerPath should
+          matchRegexFullyWithMessage(AbsolutePathPattern, "containerPath must not starts with \"/\"")
```

Comment:
is `containerPath` really needed in these messages? doesn't accord add that in automatically when it parses the object hiearchy (object == the subject of validation)?